app/controllers/shelters_controller.rb

```diff
@@ -6,4 +6,20 @@ def index
     def show
         @shelter = Shelter.find(params[:id])
     end
+
+    def new
+
+    end
+
+    def create
+        shelter = Shelter.new({
+        name: params[:shelter][:name],
+        address: params[:shelter][:address],
+        city: params[:shelter][:city],
+        state: params[:shelter][:state],
+        zip: params[:shelter][:zip]
+        })
+        shelter.save
+        redirect_to "/shelters"
```

Comment:
This section then redirects the user back to the all shelters page so they can see that their shelter was added to the list of all shelters.